EC-CUBE/ec-cube
# 【PC】【商品一覧画面】IE11にて商品一覧画面時表示時にパンくずリストのレイアウト崩れが発生している

## 概要(Overview・Refs Issue)

 - not breaking on breakcum in IE11

## 方針(Policy)



## 実装に関する補足(Appendix)


## テスト（Test)
- open with IE 11
- check not break on breadcum
